.gitignore

```diff
@@ -7,6 +7,7 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+package-lock.json
```

Comment:
this is supposed to go in version control if using npm>=5

don't think there's any harm to it